### PR TITLE
Move some tests from Travis CI to Github Actions

### PR DESCRIPTION
**Pull request by henrywang**

### CODE REVIEW

File: .github/workflows/eslint-styleline.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: eslint-stylelint
+
+on:
+  pull_request:
+    branches:
+      - master
+      - rhel-8
```
Comment (martinpitt) [inline .github/workflows/eslint-styleline.yml:7]: Any reason why you are so picky here? `on: [pull_request]` should be quite enough, and then also cover future stable branches?

File: .github/workflows/rpm-build.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+jobs:
+  rpm-build:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run rpm-build test
+        run: make test_rpmbuild
```
Comment (martinpitt) [inline .github/workflows/rpm-build.yml:17]: Does that invoke the container by itself? I don't see a `container:` in the workflow, but then again Travis didn't do that either.

**Review by martinpitt — DISMISSED** [review]
Nice, thank you! One proposed simplification and one question, but this can also be refined later.

**Comment by henrywang** [comment]
Thanks @martinpitt! :+1:  All update!

**Comment by martinpitt** [comment]
@henrywang : Your call of course, but I found the `make test_rpmbuild` better. I didn't realize at first that this calls `sudo docker run ...` by itself, which is fine -- it makes it much easier to run that test locally. Of course at some point the Makefile should be changed to work with podman, and unprivileged :) (But that's for another time).

**Review by martinpitt — APPROVED** [review]
(but good enough for sure)

**Pull request merged**